manual/english/Searching/Percolate_query.md

```diff
@@ -2342,6 +2415,68 @@ ERROR 1064 (42000): Bad JSON objects in strings: 2
 +---------------------+
 ```
 
+<!-- intro -->
+JSON:
+<!-- request JSON -->
+
+```JSON
+POST /sql?mode=raw -d "CALL PQ('products', ('{"title": "nice pair of shoes", "color": "blue"}', '{"title": "beautiful bag"}'))"
+
+POST /sql?mode=raw -d "CALL PQ('products', ('{"title": "nice pair of shoes", "color": "blue"}', '{"title": "beautiful bag}'));
+
+POST /sql?mode=raw -d "CALL PQ('products', ('{"title": "nice pair of shoes", "color": "blue"}', '{"title": "beautiful bag}'), 1 as skip_bad_json)"
```

Comment:
Can this be done not via /sql?mode=raw?

Reply:
No, Manticore doesn't support `CALL PQ` without `raw` mode.